threads/models.tsp

```diff
@@ -38,18 +45,34 @@ model ThreadObject {
   /** The identifier, which can be referenced in API endpoints. */
   id: string;
 
-  /** The object type, which is always `thread`. */
+  @doc("""
+  The object type, which is always `thread`.
+  """)
   object: "thread";
 
+  // Tool customization: _at and created are Unix encoded utcDateTime
   /** The Unix timestamp (in seconds) for when the thread was created. */
   @encode("unixTimestamp", int32)
   created_at: utcDateTime;
 
   /**
-    * Set of 16 key-value pairs that can be attached to an object. This can be useful for storing
-    * additional information about the object in a structured format. Keys can be a maximum of 64
-    * characters long and values can be a maxium of 512 characters long.
-    */
+   * Set of 16 key-value pairs that can be attached to an object. This can be useful for storing
+   * additional information about the object in a structured format. Keys can be a maximum of 64
+   * characters long and values can be a maxium of 512 characters long.
+   */
   @extension("x-oaiTypeLabel", "map")
   metadata: Record<string> | null;
-}
\ No newline at end of file
+}
+
+model ListThreadsResponse {
```

Comment:
Funny. Is there an undocumented "list threads" endpoint?

Reply:
Oh, that's interesting! Yes, I believe there must be; it's very strange that the operation doesn't show up anywhere. We should look into that, but for this purpose I think we should keep parity.